readme.md

```diff
@@ -89,15 +116,19 @@ server.addRoute({
 
 // **All code specified in the rest of the readme should be added here**
```

Comment:
This isn't going to be bolded, could put a multiline comment here instead

Reply:
Updated to multiline comment